charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -74,9 +74,12 @@ spec:
 {{ toYaml . | indent 4 }}
 {{- end }}
 {{- end }}
-    {{- if .Values.tls.broker.enabled }}
+    {{- if or .Values.tls.broker.enabled .Values.broker.extraVolumes }}
     volumes:
     {{- include "pulsar.broker.certs.volumes" . | nindent 4 }}
```

Comment:
Will there be a problem in the output of `{{- include "pulsar.broker.certs.volumes" . | nindent 4 }}` if `.Values.tls.broker.enabled` is false?